core/src/com/unciv/logic/map/tile/TileStatFunctions.kt

```diff
@@ -27,16 +27,28 @@ class TileStatFunctions(val tile: Tile) {
         localUniqueCache: LocalUniqueCache = LocalUniqueCache(false)
     ): Stats = getTileStats(tile.getCity(), observingCiv, localUniqueCache)
 
-    fun getTileStats(city: City?, observingCiv: Civilization?,
-                             localUniqueCache: LocalUniqueCache = LocalUniqueCache(false)
+    fun getTileStats(
+        city: City?, observingCiv: Civilization?,
+        localUniqueCache: LocalUniqueCache = LocalUniqueCache(false)
     ): Stats {
-        val stats = getTileStatsBreakdown(city, observingCiv, localUniqueCache).toStats()
+        val statsBreakdown = getTileStatsBreakdown(city, observingCiv, localUniqueCache)
 
-        for ((stat, value) in getTilePercentageStats(observingCiv, city, localUniqueCache)) {
-            stats[stat] *= value.toPercent()
-        }
+        val improvement = tile.getUnpillagedTileImprovement()
 
-        return stats
+        val road = if (!tile.roadIsPillaged && tile.roadStatus != RoadStatus.None)
+            tile.roadStatus.improvement(tile.ruleset)
```

Comment:
Isn't this already a function in Tile? If not then shame on us 😓

Reply:
I believe the assumption was RoadStatus was supposed to cover these areas and roads were hardcoded, so it wasn't thought that we'd need the improvement. I'm half surprised you have even just this function